ui/src/frontend/track_group_panel.ts

```diff
@@ -138,7 +138,8 @@ export class TrackGroupPanel extends Panel<Attrs> {
             (this.trackGroupState.collapsed && child !== null) ?
                 m('h2.track-subtitle', child) :
                 null),
-          selection && selection.kind === 'AREA' ?
+              m('.track-buttons', ...this.getTrackGroupActionButtons(),
+              selection && selection.kind === 'AREA' ?
```

Comment:
this line was not changed other than the indentation. Is the indentation of lines 141 and 142 ok?

Reply:
This may be a change brought on by prettier since it was reintroduced in https://github.com/android-graphics/sokatoa/pull/157.  If that's the case it wasn't conforming to Perfetto's prettier format before, but should be now.